index.html

```diff
@@ -1614,6 +1614,61 @@ <h4>LinkedDomains</h4>
 
       </section>
 
+      <section>
+        <h4>LinkedVerifiablePresentation</h4>
+        <table class="simple" style="width: 100%;">
+          <thead>
+            <tr>
+              <th>Normative Definition</th>
+              <th>JSON-LD</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>
+                <a href="https://identity.foundation/linked-vp/">Linked Verifiable Presentation</a>
+              </td>
+              <td>
+                <a href="https://identity.foundation/linked-vp/contexts/v1">Linked Verifiable Presentation</a>
+              </td>
+            </tr>
+          </tbody>
+        </table>
+
+        <pre class="example" title="Example of service and serviceEndpoint properties">
+{
+  "@context": [
+    "https://www.w3.org/ns/did/v1",
+    "https://identity.foundation/contexts/linked-vp/v1"
```

Comment:
I couldn't resolve this URL. Isn't it published yet?

Reply:
Done.